Gutenberg/config.yml

```diff
@@ -1,14 +1,4 @@
 ref:
-  # This should have either a commit or tag key.
-  # If both are set, tag will take precedence.
-  #
-  # We have automation that reads and updates this file.
-  # Leaving commented keys is discouraged, because as the automation would ignore them and the resulting updates would be noisy
-  # If you want to use a local version, please use the LOCAL_GUTENBERG environment variable when calling CocoaPods.
-  #
-  # Example:
-  #
-  #   LOCAL_GUTENBERG=../my-gutenberg-fork bundle exec pod install
-  tag: v1.102.1
+  commit: 3025c34b5015f43884b697cb0cd80826c97138ce
```

Comment:
I noticed that this reference is not pointing to the latest commit of the Gutenberg Mobile PR, the SHA should be `273a193d6c0fcb05d09e1722902c8d985f089422`. Nevertheless, since the commits pushed after this reference only introduce an E2E test fix and update the release notes, we could directly use the release Git tag once we merge the Gutenberg PRs.